image/cli/mascli/functions/install

```diff
@@ -744,18 +744,25 @@ function install() {
       --dev-mode)
         DEV_MODE=true
         ;;
+      --skip-grafana-install)
+        SKIP_GRAFANA_INSTALL=true
+        ;;
       -h|--help)
         install_help
         ;;
     esac
   done
 
-  # Determine whether Grafana is available in the cluster based on the available of the operator package
-  oc get packagemanifest grafana-operator &> /dev/null
-  if [[ "$?" == "1" ]]; then
+  if [[ "$SKIP_GRAFANA_INSTALL" == "true" ]]; then
```

Comment:
and then here we just do a check to see if `CLUSTER_MONITORING_INCLUDE_GRAFANA` is defined already, if not defined then we set it upon packagemanigest logic
```
# if CLUSTER_MONITORING_INCLUDE_GRAFANA is not defined 
if [[ -z "$CLUSTER_MONITORING_INCLUDE_GRAFANA" ]]; then
```